openpype/lib/transcoding.py

```diff
@@ -31,37 +31,9 @@
     "subimages",
 }
 
-XML_UNUSED_CHARS = {
-    "&#00;",
-    "&#01;",
-    "&#02;",
-    "&#03;",
-    "&#04;",
-    "&#05;",
-    "&#06;",
-    "&#07;",
-    "&#08;",
-    "&#11;",
-    "&#12;",
-    "&#14;",
-    "&#15;",
-    "&#16;",
-    "&#17;",
-    "&#18;",
-    "&#19;",
-    "&#20;",
-    "&#21;",
-    "&#22;",
-    "&#23;",
-    "&#24;",
-    "&#25;",
-    "&#26;",
-    "&#27;",
-    "&#28;",
-    "&#29;",
-    "&#30;",
-    "&#31;"
-}
+CHAR_REF_REGEX_DECIMAL = re.compile(r"&#[0-9]+;")
```

Comment:
you can use one regex to rule them all:
```&#x?(?P<char>[0-9a-fA-F]+);`

Reply:
I wanted to add the ranges validations...

Reply:
sure, just that one will pick all character references no matter if they are hex or decimal and you can do the range check on `m.group("char")` then. You just need to make sure it is cast properly to number.

Reply:
But I've modified it like you suggested here. I don't want be dependent on oiiotool "correctness" of xml.